web/src/RootSSHKey.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import { FileUpload } from "@patternfly/react-core";
+
+export default function RootSSHKey(props) {
```

Comment:
NP: a matter of taste, but I prefer destructuring directly here. I mean, going for `{ value, valueChange }` instead of `props`. 

Another note, for those callbaks I usually see the _convention_ **on**ValueChanged.